cmd/livecheck.rb

```diff
@@ -390,6 +394,16 @@ def self.latest_version(formula)
       match_version_map = strategy_data[:matches]
       regex = strategy_data[:regex]
 
+      if strategy_data[:messages].is_a?(Array) && match_version_map.empty?
+        puts strategy_data[:messages] unless Homebrew.args.json?
+
+        if i + 1 < urls.length
+          next
+        else
+          return status_hash(formula, "error", strategy_data[:messages])
+        end
```

Comment:
```suggestion
        next if i + 1 < urls.length
        return status_hash(formula, "error", strategy_data[:messages])
```

(feel free to modify your commit and force-push)

Reply:
I thought about doing this but `brew style` will want an empty line after the guard clause, so it ends up being:

```ruby
next if i + 1 < urls.length

return status_hash(formula, "error", strategy_data[:messages])
```

I figured this wasn't that far off from the existing if/else and was maybe less clear. However, if we move the guard underneath the `puts`, I think it's fine:

```ruby
puts strategy_data[:messages] unless Homebrew.args.json?
next if i + 1 < urls.length

return status_hash(formula, "error", strategy_data[:messages])
```

I'll go ahead and integrate this, as it is a bit tidier.